renovate.json

```diff
@@ -1,40 +1,72 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "automerge": false,
-  "baseBranches": ["main"],
+  "baseBranches": ["main", "20.0.x"],
```

Comment:
To enable this, you need to enable also add https://github.com/angular/angular/blob/main/.github/workflows/sync-angular-robot-forked-repo.yml

Reply:
Looks like this is still here